tfx/components/trainer/executor.py

```diff
@@ -58,6 +59,31 @@ def _is_chief():
   return task_type == 'chief' or (task_type == 'master' and task_index == 0)
 
 
+def _serving_model_path(working_dir: Text) -> Text:
+  """Given model artifact working directory, returns original path
+  for timestamped and named serving model exported."""
```

Comment:
less than 80 chars, otherwise you need a full doc format like  [1], same else where

[1]https://github.com/tensorflow/tfx/blob/master/tfx/components/example_gen/base_example_gen_executor.py#L175